### DISCUSSION TITLE
How to implement Custom Security Authentication in Superset

### DISCUSSION THREAD

**Discussion by Manik0917** [apache/superset#16788] [Q&A / Help]
My application is deployed using Helm chart into AKS POD. Superset is also deployed (using Docker image of Apache/Superset) into another POD with basic authentication. Dashboard is created in Superset. I would like to embed this superset dashboard with my web application using Iframe. However I am expecting my web application to auto login with configured user on Superset basic authentication to embed dashboard in my web application without showing login screen of superset. Please do share sample code to implement Custom Security Authentication using helm chart

**Comment by junlincc** [comment]
@nytai @dpgaspar may know 🙏

**Reply by Manik0917** [reply]
@nytai @dpgaspar Could you please provide the solution asap for me to implement it?
